ee/server/service/vpp.go

```diff
@@ -369,17 +383,17 @@ func (svc *Service) AddAppStoreApp(ctx context.Context, teamID *uint, appID flee
 	return nil
 }
 
-func getVPPAppsMetadata(ctx context.Context, ids []fleet.VPPAppID) ([]*fleet.VPPApp, error) {
+func getVPPAppsMetadata(ctx context.Context, ids []fleet.VPPAppTeam) ([]*fleet.VPPApp, error) {
 	var apps []*fleet.VPPApp
 
 	// Map of adamID to platform.
```

Comment:
Nit, but just to help understand the nested map - "to platform then whether it's available as self-service", I think?

Reply:
I don't quite understand the question, sorry

Reply:
I updated the comment, I think that's what you were getting at right?